package.json

```diff
@@ -147,15 +147,15 @@
                 "linux": "Z",
                 "key": "Z",
                 "when": "notebookEditorFocused && !inputFocus && notebookViewType == jupyter-notebook && config.jupyter.enableKeyboardShortcuts",
-                "command": "undo"
+                "command": "jupyter.notebookeditor.keybind.undo"
             },
             {
                 "mac": "S",
                 "win": "S",
                 "linux": "S",
                 "key": "S",
                 "when": "notebookEditorFocused && !inputFocus && notebookViewType == jupyter-notebook && config.jupyter.enableKeyboardShortcuts",
-                "command": "workbench.action.files.save"
+                "command": "jupyter.notebookeditor.keybind.save"
```

Comment:
We should file a bug against vS Code, 
the `when` clause makes this PR unnecessary, (we have a clause to ensure this keybinding does not work when a cell has input focus).

Reply:
I.e. we're adding a work around for a VS Code bug & I'd prefer VS Code fixes it long term.